workflows/sars-cov-2-variant-calling/sars-cov-2-consensus-from-variation/consensus.ga

```diff
@@ -0,0 +1,1735 @@
+{
```

Comment:
@wm75 I think this needs to be called consensus-from-variation.ga for planemo to pick up the test (or rename the test file to consensus-test.yml).

Reply:
I see, thanks.

Reply:
Alright, that looks better:
```
Applying linter structure... WARNING
.. WARNING: Workflow contained output without a label
Applying linter tests... CHECK
.. CHECK: Tests appear structurally correct for workflows/sars-cov-2-variant-calling/sars-cov-2-consensus-from-variation/consensus-from-variation.ga
```